packages/db/src/meta/pouch/adapters/base.ts

```diff
@@ -81,10 +81,23 @@ export abstract class Databases<C extends Collections> implements Workspace<C> {
     const log = debug.extend(`${collectionName}:all`);
     log("Fetching all...");
 
-    const result = await this.find<N>(collectionName, { selector: {} });
+    try {
+      const { rows }: any = await this.collections[collectionName].allDocs({
+        include_docs: true
```

Comment:
I am sure this is necessary, but seems so redundant, haha.

Reply:
By default, `allDocs` just returns `_id` and `_rev`, apparently